collect_app/src/main/res/layout/url_widget_answer.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/url_button"
+        style="@style/Widget.Collect.Button.Custom"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/margin_small"
+        android:text="@string/open_url"
+        android:layout_marginStart="@dimen/margin_standard"
+        android:layout_marginEnd="@dimen/margin_standard"
+        android:layout_marginTop="@dimen/margin_extra_small"
+        />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/url_answer_text"
+        android:layout_width="match_parent"
```

Comment:
Can this also be a style so it's easy to make consistent across widget types?

Reply:
Yeah actually this should have a `TextAppearance` style. Probably `TextAppearance.Collect.Body1`.

Reply:
Yes, I have seen that style probably in `ProgressDialogFragment`. I will use that.

Reply:
What about also including padding, height, color, etc? I think we will want to change how answers are displayed and it'd be great to be able to do that in a single place for all widgets.

Reply:
Yes, I think we can do that, for TextView and Buttons that we use, as we have the same style for them for many widgets.